RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -2051,7 +2053,7 @@ public boolean isEmpty() {
    *
    * @return the iterator
    */
-  @Override
+
   public Iterator<Integer> iterator() {
```

Comment:
I feel the method should be dropped alltogether, or at least marked `@Deprecated`